src/main/java/mods/railcraft/common/util/misc/Game.java

```diff
@@ -33,17 +32,13 @@
     public static final boolean BUKKIT;
 
     static {
-        boolean worldFound = false;
-        boolean worldObjFound = false;
+        boolean obfuscated;
         try {
-            worldFound = Entity.class.getDeclaredField("world") != null;
-        } catch (NoSuchFieldException | SecurityException ignored) {
+            obfuscated = MinecraftServer.class.getDeclaredField("worldServers") == null;
+        } catch (NoSuchFieldException absence) {
+            obfuscated = true;
         }
-        try {
-            worldObjFound = Entity.class.getDeclaredField("worldObj") != null;
-        } catch (NoSuchFieldException | SecurityException ignored) {
-        }
-        OBFUSCATED = !worldFound && !worldObjFound;
+        OBFUSCATED = obfuscated;
```

Comment:
Cuz this is done after merging. You should consider this as a merge conflict, and I will fix it too,